src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ProjectCard.kt

```diff
@@ -0,0 +1,30 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import javafx.geometry.Pos
+import javafx.scene.layout.VBox
+import org.wycliffeassociates.otter.common.data.model.Project
+import org.wycliffeassociates.otter.common.ui.widgets.IProjectCard
+import tornadofx.*
+import tornadofx.Stylesheet.Companion.root
+
+class ProjectCard(project : Project, openProject: () -> Unit) : VBox(), IProjectCard{
```

Comment:
Let's look at how click events are usually handled in UI controls for JavaFX- passing in a lambda doesn't seem to be consistent with how you use anything else.